src/kernel-install/90-loaderentry.install.in

```diff
@@ -101,6 +101,11 @@ if [ -f "$TRIES_FILE" ]; then
         echo "$TRIES_FILE does not contain an integer." >&2
         exit 1
     fi
+    if [ -f "$LOADER_ENTRY" ]; then
+        [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && \
+            echo "Removing previous loader entry '$LOADER_ENTRY' without boot counting." >&2
+        rm -f "$LOADER_ENTRY"
```

Comment:
this should not just remove the one without boot counting, but also with *other* values of boot counting.

hence something like this too:

```sh
rm -f "$LOADER_ENTRY" "${LOADER_ENTRY%.conf}+"*.conf
```

or so?

Reply:
Changed it like that